feat: support for inter-services calls (#1273)
lightbend/kalix-jvm-sdk: docs/src/modules/spring/pages/call-another-service.adoc

= Calling other services

== Kalix services

Calling other Kalix services in the same project from an Action is done by invoking them using a https://docs.spring.io/spring-framework/docs/5.0.13.RELEASE/spring-framework-reference/web-reactive.html#webflux-client[`Spring WebFlux WebClient`]. The service is identified by the name it has been deployed. Kalix takes care of routing requests to the service and keeping the data safe by encrypting the connection for you.

In this sample we will make an action that does a call to the xref:value-entity.adoc[Value Entity Counter] service, deployed with the service name "counter."

> The Kalix Spring SDK provides a utility class calls `WebClientProvider` that can provide already configured `WebClients` for other Kalix services deployed on the same Kalix project.

Reviewer: "already configured" sounded a bit weird..

```suggestion
The Kalix Spring SDK provides a utility class `WebClientProvider` that can provide previously configured `WebClient`s to reach other Kalix services deployed on the same Kalix project.
```
